ledger/src/entry.rs

```diff
@@ -162,15 +163,73 @@ pub fn next_hash(start_hash: &Hash, num_hashes: u64, transactions: &[Transaction
     }
 }
 
+pub struct EntryVerifyState {
+    thread_h: Option<JoinHandle<()>>,
+    hashes: Option<Arc<Mutex<Vec<Hash>>>>,
+    verified: bool,
+    tx_hashes: Vec<Option<Hash>>,
+    start: Instant,
+    gpu_wait: Instant,
+}
+
+impl EntryVerifyState {
+    pub fn finish_verify(&mut self, entries: &[Entry]) -> bool {
+        if self.hashes.is_some() {
+            self.thread_h.take().unwrap().join().unwrap();
+            inc_new_counter_warn!(
```

Comment:
If replay takes significantly longer than the gpu verify, it seems this counter will skew the metric for how long the GPU computation took b/c this counter always reports a time at least as long as the replay time

Reply:
True, probably should just remove it.

Reply:
Or maybe the gpu thread can do the timing and return it.

Reply:
Fixed this.